### PR TITLE
Try to fix core coverage issue

### PR DESCRIPTION
**Pull request by dafortin**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* **Fixed** coverage

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ install:
 - bower install
 script:
 - npm run lint
-- ember try:one $EMBER_TRY_SCENARIO --- ember test
+- ember try:one $EMBER_TRY_SCENARIO --- COVERAGE=true ember test
```
Comment (rox163) [inline .travis.yml:30]: yay!

**Comment by rox163** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/325/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

**Comment by sglanzer-fire** [comment]
So freakin' happy about this!  Thank you!  Do we have an issue raised to address the `skip`?

**Comment by dafortin** [comment]
I will raise one good point

**Comment by dafortin** [comment]
https://github.com/ciena-frost/ember-frost-core/issues/327

**Pull request merged**